### PR TITLE
Fix Cookie Not Updating dB queires

### PR DESCRIPTION
**Pull request by MM-coder**
The cookie elif didn't set `dailyRewardClaimed` and `dailyRewardStreak` hence people would lose their streaks

PRing to master due to priority of fix.

### CODE REVIEW

File: commands/daily.py [hunk 1]
```diff
@@ -78,6 +78,10 @@ def check(m):
                     cookies_no = sql.db_query("SELECT cookiesReceived FROM Members WHERE UserID = %s" % (str(ctx.author.id)))[0][0]
                     cookies_no = cookies_no + 3
                     sql.execute_query("UPDATE Members SET cookiesReceived = %s WHERE UserID = %s" % (str(cookies_no), str(ctx.author.id)))
+                    next_advent =  current_time + ((60 * 60) * 20)
+                    sql.execute_query("UPDATE Members SET dailyRewardClaimed = %s WHERE UserID = %s " % (str(next_advent), str(ctx.author.id)))
+                    new_streak = streak + 1
+                    sql.execute_query("UPDATE Members SET dailyRewardStreak = %s WHERE UserID = %s " % (str(new_streak), str(ctx.author.id)))
```
Comment (FrankKeller) [inline commands/daily.py:84]: Code Duplication? While we're at it we might as well fix the mess that is this command.

**Review by FrankKeller — CHANGES_REQUESTED** [review]
Code Duplication. Would be better to have the streak and claimed values set in their own routine rather than duplicating it at the end of each option

**Comment by FrankKeller** [comment]
Also this should go to wilfred-360 which will then be merged into master, this is so the latest release (360) isn't out of sync with master. Also allows us to integration test before pushing it to master. That said, I would recommend pushing this to 361 as that is being released beginning of November

**Comment by MattW-source** [comment]
Target 361. Also in agreement with @FrankKeller to move the code into a method and call that method rather than having lots of code duplication.

**Pull request closed without merging**